cumulusci/core/flowrunner.py

```diff
@@ -244,6 +261,7 @@ def run(self, org_config):
             line = "{} ({})".format(line, self.name)
         self._rule()
         self.logger.info(line)
+        self.logger.info("Flow Description: {}".format(self.flow_config.description))
```

Comment:
kinda would prefer just a ** or something instead of "Flow Description:" to reduce line text?